keyboards/ut472/rules.mk

```diff
@@ -67,3 +67,4 @@ BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 RGBLIGHT_ENABLE = yes
+#TAP_DANCE_ENABLE = yes      # Enable tap dance feature
```

Comment:
This should not be commented out by default, but rather set to `no` (or removed entirely). Keymaps can still override this in their own rules.mk files.